src/resources/Restoration/BC.ts

```diff
@@ -41,8 +41,7 @@ export const RestorationResourceBc: RestorationResourceIF = {
       certifyStatements: [],
       certifyClause: `Note: It is an offence to make or assist in making a false or misleading
         statement in a record filed under the BC Corporations Act. A person who commits this
-        offence is subject to a maximum fine of $5,000.`,
-      entityDisplay: 'BC Corporations Act'
```

Comment:
What did the old certify block look like with this value?

Reply:
![image](https://github.com/user-attachments/assets/94e2355f-06ac-40de-a3d3-fdd746ace65f)

Reply:
Thanks.

Yeah, that doesn't make sense to me. Much better with "business" :)

Resolving.